.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=hca
```

Comment:
Oh, I'm actually fine with leaving this as is. I'm just not very familiar with docker-compose, so I thought for a moment the image would be called "hca".